packages/@webex/internal-plugin-presence/src/presence.js

```diff
@@ -45,6 +45,20 @@ const Presence = WebexPlugin.extend({
     });
   },
 
+  /**
+   * Initializes the presence worker.
+   * @returns {undefined}
+   */
+  initializeWorker() {
+    if (this.webex.ready) {
+      this.worker.initialize(this.webex);
+    } else {
+      this.webex.once('ready', () => {
+        this.worker.initialize(this.webex);
+      });
+    }
+  },
```

Comment:
Cantina is going to set config.initializeWorker to false and then initialize the worker later.